packages/fetch-and-compile/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+    "extends": ["../../.eslintrc.package.json"]
+  }
```

Comment:
Sorry for more stuff that should be taken care of by `prettier`, but -- oh, hang on, this is a JSON file, it wouldn't be taken care of by `prettier` even if it were working! :P  Anyway, sorry for more of these tiny style nitpicks, but something's gone wrong with your indentation here.

Btw, you seem to be using an editor that doesn't insert newline at end of file (as indicated in the GitHub display with that big red minus-in-a-circle), so you may want to add those manually (until `prettier` is working again... although for this file, as it's JSON, you'd have to do it manually anyway. :P )

Reply:
Okay I'll take a look at this and comment once it's fixed. Also my apologies for not replying to the previous comments before requesting the next round. I fixed all the things you mentioned beside the issue relating to importing modules as I was waiting to hear back on that one before fixing. I just didn't comment on them here before pushing the changes cause I didn't know that was part of the workflow but now that I know I will be sure to do that in the future. Thanks for bringing that up and thanks for reviewing this again!

Reply:
To be clear, the idea isn't that you have to comment on *everything*; just that, you know, for each thing it should be made clear what's going on.  If it's a simple thing you can just directly implement, well, you can just do that; not much to say.  (Although it's still useful to hit the thumbs-up button to acknowledge the comment.)  But if you're leaving something unaltered for now, or if the code changes are complex or wouldn't be obvious ot the reviewer... you should use this space to make sure the reviewer understands the current status of things!

Reply:
Okay sweet thanks for the clarity on that :)